bodhi/server/consumers/composer.py

```diff
@@ -642,7 +622,7 @@ def expire_buildroot_overrides(self):
 
     def remove_pending_tags(self):
         """Remove all pending tags from the updates."""
-        self.log.debug("Removing pending tags from builds")
+        log.debug("Removing pending tags from builds")
```

Comment:
This pull request isn't really about changing the way we reference our logger. I'm not opposed to the change in general, but it would be better to put it in a separate pull request, since it is a separate concern.